master/reference/calicoctl/resources/ippool.md

```diff
@@ -70,6 +67,18 @@ Increasing the block size from the default (e.g., using `24` for IPv4 to give 25
 
 Reducing the block size from the default (e.g., using `28` for IPv4 to give 16 addresses per block) means more blocks per host and therefore potentially more routes. This can be beneficial if it allows the blocks to be more fairly distributed amongst the hosts.
 
+#### Node Selector
+
+{% include {{page.version}}/selectors.md %}
+
+For details on configuring IP pool node selectors on your deployment, please read the
```

Comment:
I'd nix the "on your deployment" bit here - I think it's assumed context, and might be confused with the kubernetes "Deployment" concept.